components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report trigerred errors,
+    legacy tests and usage of deprecated code.
```

Comment:
>The summary only talks about the deprecation utilities, but it provides other testing utilities as well

Well before you raised the point on the `ClockMock` I though that was the case to be fair...

Do we want to mention the bootstrap file? It's more an implementation detail than anything else right? So should we really talk about it in the introduction?